Update eleventy version to the latest version (resolves #87)

chosww opened the pull request:
When the eleventy version was updated to the latest, there was a breaking change, and it was fixed by wrapping files to include in templates with quotation marks.

greatislander commented on package.json:
```diff
@@ -20,7 +20,7 @@
     "dependencies": {
-        "@11ty/eleventy": "0.11.0",
+        "@11ty/eleventy": "2.0.1",
         "@11ty/eleventy-plugin-rss": "1.0.7",
         "@11ty/eleventy-plugin-syntaxhighlight": "3.0.1"
```
Are updates needed for these packages?

chosww commented:
@greatislander I removed gpii-grunt-lint-all as it is deprecated and added fluid-lint-all. This introduces some issues with markdownlint and I was wondering if that is something you are aware of?

chosww commented:
> @greatislander I removed gpii-grunt-lint-all as it is deprecated and added fluid-lint-all. This introduces some issues with markdownlint and I was wondering if that is something you are aware of?

fluid-lint-all uses markdownlint-config-fluid, and markdownlint-config-fluid uses vulnerable version of markdownlint. Tony has opened an issue for dependency update in markdownlint-config-fluid repo: https://github.com/fluid-project/markdownlint-config-fluid/issues/5

chosww commented on package.json:
```diff
@@ -20,20 +20,23 @@
-        "grunt": "1.1.0",
+        "fluid-lint-all": "1.2.9",
+        "grunt": "1.6.1",
         "grunt-contrib-clean": "2.0.0",
         "grunt-contrib-copy": "1.0.0",
```
Thanks Ned, when I run depcheck this is the result I get:

Unused devDependencies
* debug
* fluid-lint-all
* grunt-contrib-clean
* infusion
* sanitize-html
* sanitize.css
Missing dependencies
* eslint-config-fluid: ./.eslintrc.json
* gpii-grunt-lint-all: ./Gruntfile.js
* jquery: ./src/lib/infusion/infusion-all.js

seems like grunt-contrib-copy is used somewhere?

chosww replied:
Should we also remove other unused dependencies from this report? Also, the two missing dependencies need to be installed, eslint-config-fluid and jquery.

greatislander commented:
@chosww Let's add a `netlify.toml` file like this one: https://github.com/codelearncreate/weavly.org/blob/main/netlify.toml

greatislander commented on package.json:
```diff
@@ -20,24 +20,22 @@
+        "npm": "9.7.2"
     },
     "devDependencies": {
         "@tbranyen/jsdom": "13.0.0",
```
Let's replace this with linkedom. See: https://github.com/greatislander/trivet/commit/ebd719ee5374b34946d4178c81a156fc12caaa9f

greatislander commented on package.json:
```diff
@@ -20,24 +20,22 @@
     },
     "devDependencies": {
         "@tbranyen/jsdom": "13.0.0",
         "debug": "4.1.1",
```
Where is this used?

greatislander commented on package.json:
```diff
@@ -20,24 +20,22 @@
         "rimraf": "3.0.2",
-        "sanitize-html": "1.24.0",
-        "sanitize.css": "11.0.1",
         "slugify": "1.4.0"
```
This can be replaced by requiring `@sindresorhus/slugify` (an Eleventy dependency) wherever it's used.

chosww replied:
After I changed it to the suggested module, I am getting an error about require() -- "Must use import to load ES Module: /Users/danielcho/Desktop/fluidproject.org/node_modules/@sindresorhus/slugify/index.js
[11ty] require() of ES modules is not supported."

greatislander replied:
Did you install `@sindrehorhus/slugify`? You don't need to add it as a dependency— it's already required by Eleventy in the older (non-module) version.

chosww replied:
Don't I have to require it on a file where slugify is used?

greatislander commented on package.json:
```diff
@@ -20,24 +20,22 @@
+        "@11ty/eleventy": "2.0.1",
+        "@11ty/eleventy-plugin-rss": "1.2.0",
+        "@11ty/eleventy-plugin-syntaxhighlight": "5.0.0",
+        "install": "0.13.0",
+        "npm": "9.7.2"
```
I think these were added by mistake :)
